backend/data/src/test/kotlin/io/tolgee/unit/formats/po/in/PoToICUConverterTest.kt

```diff
@@ -39,6 +39,50 @@ class PoToICUConverterTest {
     )
   }
 
+  @Test
+  fun testPhpPluralsWithHashtags() {
```

Comment:
Can you please name the tests conventionally?
```suggestion
  fun `php plurals with hashtags works`() {
```

Reply:
Other tests in this file are named like this. Should I change all of them?